pkg/health/client/client.go

```diff
@@ -234,28 +253,84 @@ func GetHostPrimaryAddress(node *models.NodeStatus) *models.PathStatus {
 	return node.Host.PrimaryAddress
 }
 
+// GetHostSecondaryAddresses returns the secondary host addresses (if any)
+func GetHostSecondaryAddresses(node *models.NodeStatus) []*models.PathStatus {
+	if node.Host == nil {
+		return nil
+	}
+
+	return node.Host.SecondaryAddresses
+}
+
+// GetAllHostAddresses returns a list of all addresses (primary and any
+// and any secondary) for the host of a given node. If node.Host is nil,
+// returns nil.
+func GetAllHostAddresses(node *models.NodeStatus) []*models.PathStatus {
+	if node.Host == nil {
+		return nil
+	}
+
+	return append([]*models.PathStatus{node.Host.PrimaryAddress}, node.Host.SecondaryAddresses...)
+}
+
+// GetEndpointPrimaryAddress returns the PrimaryAddress for the health endpoint
+// within node. If node.Host is nil, returns nil.
```

Comment:
```suggestion
// within node. If node.HealthEndpoint is nil, returns nil.
```

(Although given that this is clearly stated in code two lines down I don't see the value in this comment at all)